### PR TITLE
chore: update readme examples to webpack 2 syntax

### PR DESCRIPTION
**Pull request by AvraamMavridis**
Minor update in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ or specify the `minimize` query in your `webpack.conf.js`
 module: {
   loaders: [{
```
Comment (michael-ciniawsky) [inline README.md]: `loaders` => `rules`

Comment (michael-ciniawsky) [inline README.md]: `loader` => `use`

Comment (michael-ciniawsky) [inline README.md]: `query` => `options`

Comment (AvraamMavridis) [inline README.md]: sure

**Comment by michael-ciniawsky** [comment]
@webpack-contrib/admin-team some merge please :)

**Comment by michael-ciniawsky** [comment]
@d3viant0ne Thx :)

**Pull request merged**